### DISCUSSION TITLE
How to set height/width through command line?

### DISCUSSION THREAD

**Discussion by longguzzz** [wezterm/wezterm#2598] [Q&A] [answered]
I am trying to set wezterm's size when using neovim to edit in different filetype. My plan is sending command from neovim to outside wezterm, and it needs command to change the size of wezterm.

I know [initial_rows](https://wezfurlong.org/wezterm/config/lua/config/initial_rows.html#initial_rows--24) and [initial_cols = 80](https://wezfurlong.org/wezterm/config/lua/config/initial_cols.html#initial_cols--80), but I don't know whether these is a dynamic way to set height/width. Is there any method?

**Comment by wez** [comment, marked as answer]
Do you mean that you want to resize the existing window?
wezterm doesn't have any functions that allow that today.

**Reply by dytavan** [reply]
I have a similar question how can we set the initial rows and column ?
I'm new with your term an lua is a bit mistery for me, I know initial_rows and columns but where to put it ???
Thanks

**Reply by wez** [reply]
[david's question has been asked and answered in https://github.com/wez/wezterm/discussions/2809]
